packages/fx-core/src/plugins/resource/bot/azureOps.ts

```diff
@@ -132,17 +136,42 @@ export class AzureOperations {
     zipDeployEndpoint: string,
     zipBuffer: Buffer,
     config: any
-  ): Promise<void> {
+  ): Promise<string> {
     let res = undefined;
     try {
       res = await axios.post(zipDeployEndpoint, zipBuffer, config);
     } catch (e) {
       throw new ZipDeployError(e);
     }
 
-    if (!res || !utils.isHttpCodeOkOrCreated(res?.status)) {
+    if (!res || !utils.isHttpCodeAccepted(res?.status)) {
       throw new ZipDeployError();
     }
+
+    return res.headers.location;
+  }
+
+  public static async CheckDeployStatus(location: string, config: any): Promise<void> {
+    let res = undefined;
+    for (let i = 0; i < DeployStatus.RETRY_TIMES; ++i) {
+      try {
+        res = await axios.get(location, config);
```

Comment:
Is this `axios` the default instance?
Creating an axios instance for deployment only to avoid unexpected default settings (auth/header/proxy...).